src/Microsoft.Azure.WebJobs.Host/Config/DirectInvokeString.cs

```diff
@@ -14,5 +14,7 @@ public DirectInvokeString(string value)
         {
             this.Value = value;
         }
+
+        public static DirectInvokeString None = new DirectInvokeString("???");
```

Comment:
Would it make more sense for this to be `string.Empty`? Although rare, "???" is still a viable option as an invoke string so it may be a little confusing.

Reply:
The invoke string is displayed in the dashboard UI, so seeing ??? would be more self-explanatory. This is a tactical concession - handling string.empty may require dashboard changes.